### PR TITLE
update README.md

### LINKED ISSUE #4: Edit README.md
As described by https://docs.google.com/spreadsheets/d/1pEIYt8chf-HVIesujlrcKw-m42RY__Ho4bKT1kjmg9Y/edit?usp=sharing

### PR DESCRIPTION
**Pull request by isabelizimm**
add overall demo context, breakdown of tools used, about the environment, how to contribute, and contact sections

closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
-project-template
+Meteor Demo
 ==============================
 
-template for the team to use
+This repository has materials for a hands-on tutorial to demonstrate tools developed by Red Hat's Artificial Intelligence Center of Excellence to aide Data Scientists and DevOps Engineers by automating many tasks involved in building an end-to-end machine learning application.
```
Comment (oindrillac) [inline README.md]:
```suggestion
This repository has materials for a hands-on tutorial to demonstrate tools developed by Red Hat's Artificial Intelligence Center of Excellence to aid Data Scientists and DevOps Engineers by automating many tasks involved in building an end-to-end machine learning application. 
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+
+This tutorial highlights the capabilities of [Project Meteor](https://github.com/AICoE/meteor), which is a web application that converts users' GitHub repositories into [JupyterBook](https://jupyterbook.org/intro.html) environments. The demo features a "hello world" MNIST machine learning application, but the focus is on the integration of different tools for techniques such as dependency management, model serving, and others detailed below. Overall, this demo highlights the fact that Data Scientists benefit from many DevOps practices. 
+
+## Tools
+
+There are a variety of tools that are highlighted in this tutorial. The main focus is on the tools below:
+
+- [Jupyterlab](https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html) is used to build Jupyter notebooks, text editors, and other components.
+- [Elyra](https://github.com/elyra-ai/elyra) supplies a set of extensions to JupterLab notebooks to support AI projects
```
Comment (oindrillac) [inline README.md:13]:
```suggestion
- [Elyra](https://github.com/elyra-ai/elyra) supplies a set of extensions to JupyterLab notebooks to support AI projects
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+- [Thoth](https://thoth-station.ninja/) is used to manage dependencies and keep tutorials up to date with the latest releases.
+- [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/overview/pipelines-overview/) are used to schedule machine learning workflows via a user interface located within the JupyterLab environment.
+- [Seldon Core](https://docs.seldon.io/projects/seldon-core/en/latest/) deploys machine learning models as REST/GRPC microservices.
+- [Grafana](https://grafana.com/grafana/) is a dashboarding tool to observe data. In this case, it is used to monitor model health.
+
+## About the Evironment
```
Comment (oindrillac) [inline README.md:19]:
```suggestion
## About the Environment
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,24 @@
...
+
+The Operate First open source production OpenShift environment currently hosts Project Meteor and this demo. Operate First looks to open-source operations on community managed clusters. To learn more about Operate First, visit the [website](https://www.operate-first.cloud/) or GitHub [community](https://github.com/operate-first).
 
 Project Organization
```
Comment (oindrillac) [inline README.md:23]: Should we remove this default cookie-cutter structure and just keep the line which says this is based off cookie cutter. Eventually when we build a "meteor template", we can highlight that here?

**Review by oindrillac — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (isabelizimm) [reply README.md]: I think that makes sense, updated :)

Comment (oindrillac) [reply README.md]: Thanks! 🚀

**Review by oindrillac — APPROVED** [review]
/lgtm

**Pull request closed without merging**